tools/windows-node-installer/pkg/types/types.go

```diff
@@ -4,6 +4,13 @@ package types
 // sub-package
 // TODO: Move every cloud provider types here
 
+const (
+	// KubeletPort no that will be opened on the windows node via firewall
+	KubeletPort = "10250"
+	// firewallCmdName is the firewall rule name to open the Kubeletport
+	FirewallCmdName = "OpenKubeletPort"
```

Comment:
This is not the firewall command but the name of the firewall rule. `FirewallRuleName`